docs/README.md

```diff
@@ -0,0 +1,624 @@
+Welcome to the *AIM API - Alpha*!
+
+# Alpha Notice
+
+The AIM API is in an "alpha stage" to gather customer feedback. While in alpha, the API may change in backwards incompatible ways to accommodate for fixes and additions. Breaking changes will be communicated to the primary API contact at least 5 business days in advance.
+
+TSG reserves the right to determine what constitutes a breaking changes. A definition of "breaking changes" will be made available before formal release.
```

Comment:
Al:
>Isn’t breaking Changes as a term already
defined, doesn’t this mean the details of each Breaking Change if there is more than one.

Reply:
Yes - breaking changes are an entire class of situations/problems. What I'm trying to say here is that we're not going to enumerate every possible type of breaking change _yet_ (of which there are many), but we'll want a more formal definition/enumeration/examples before beta/public release.

Something like this [Google API compatibility page](https://cloud.google.com/apis/design/compatibility) is what we'll want before release (note the language: "breaking and non-breaking changes"). We're ready to dive into as much detail as that page shows - that page is probably longer than this whole document and handles just these two sentences).